src/open_inwoner/utils/decorators.py

```diff
@@ -1,26 +1,61 @@
 import inspect
 import logging
+import re
+import uuid
 from collections.abc import Callable
 from functools import wraps
 from typing import TypeVar
 
-from django.core.cache import caches
+from django.core.cache import BaseCache, caches
 
 logger = logging.getLogger(__name__)
 
 
 RT = TypeVar("RT")
 
 
-def cache(key: str, alias: str = "default", **set_options):
+def _map_cache_key_instance_attrs_to_placeholders(key):
```

Comment:
Could use a brief docstring explaining the need for this mapping due to supporting multiple backends with different endpoints.